src/closure.js

```diff
@@ -1,30 +1,37 @@
 // Complete the following functions.
 
 const counter = () => {
-  // Return a function that when invoked increments and returns a counter variable.
-  // Example: const newCounter = counter();
-  // newCounter(); // 1
-  // newCounter(); // 2
+  let ct = 0;
```

Comment:
People have different opinions on this but I prefer more descriptive variable names like `count` over `ct`